### PR TITLE
Updated with the latest changes (#15)

### PR DESCRIPTION
**Pull request by vagrantism**

### CODE REVIEW

File: nl2sql/datasets/base.py
Comment (solidate) [inline nl2sql/datasets/base.py:1]: What's the benefit of new changes over previous implementation of table descriptor implementation? @vagrantism

**Review by solidate — CHANGES_REQUESTED** [review]
Remove the .vscode folder

File: nl2sql/datasets/base.py
Comment (vagrantism) [reply nl2sql/datasets/base.py:1]: Generating descriptions post-hoc ensures any filtered-out column is also removed from the descriptions. Earlier, doing this pre-hoc resulted in a bug where column filtering would have no effect on the descriptions. @tanyarw is the original author of this fix.

**Comment by vagrantism** [comment]
> Remove the .vscode folder

@solidate The .vscode folder contains some basic (but important) linting configs that we would want our contributors to follow.

**Comment by solidate** [comment]
> > Remove the .vscode folder
> 
> @solidate The .vscode folder contains some basic (but important) linting configs that we would want our contributors to follow.

All contributor not necessarily use vscode. The lint guide should be part of CONTRIBUTING.md
WDYT @vagrantism

**Comment by vagrantism** [comment]
> > > Remove the .vscode folder
> > 
> > 
> > @solidate The .vscode folder contains some basic (but important) linting configs that we would want our contributors to follow.
> 
> All contributor not necessarily use vscode. The lint guide should be part of CONTRIBUTING.md WDYT @vagrantism

Done, PTAL

**Review by solidate — APPROVED** [review]
LGTM

**Review by tanyarw — APPROVED** [review]
LGTM

**Pull request merged**